pkg/apps/sonarr.go

```diff
@@ -1112,6 +1115,82 @@ func sonarrDeleteAllCustomFormats(req *http.Request) (int, interface{}) {
 	}
 }
 
+// @Description  Returns all Import Lists from Sonarr.
+// @Summary      Get Sonarr Import Lists
+// @Tags         Sonarr
+// @Produce      json
+// @Param        instance  path   int64  true  "instance ID"
+// @Success      200  {object} apps.Respond.apiResponse{message=[]sonarr.ImportListOutput}  "import list list"
```

Comment:
"list of import lists"

Reply:
This was copied from one of the other existing apps so it would need to be changed on more than just this one